### PR TITLE
chore: merge tagged release without e2e tests/beta

### PR DESCRIPTION
**Pull request by sobolk**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by akshbhu — APPROVED** [review]

File: packages/amplify-category-storage/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/amplify-category-storage",
-  "version": "4.0.0-cdkv2.2",
+  "version": "4.0.1-beta.0",
```
Comment (awsluja) [inline packages/amplify-category-storage/package.json]: is storage the only category that is in beta?

Comment (sobolk) [reply packages/amplify-category-storage/package.json]: yeah, there is this remnant on npm
image

Comment (akshbhu) [reply packages/amplify-category-storage/package.json]: it will fail again when releasing next patch version

Comment (sobolk) [reply packages/amplify-category-storage/package.json]: No, we went to `4.0.1-beta.1` to avoid that.

Comment (sobolk) [reply packages/amplify-category-storage/package.json]: https://www.npmjs.com/package/@aws-amplify/amplify-category-storage/v/4.0.1-beta.1

**Review by awsluja — APPROVED** [review]

**Pull request merged**